pkg/webhook/image_swapper.go

```diff
@@ -172,8 +178,7 @@ func (p *ImageSwapper) Mutate(ctx context.Context, ar *kwhmodel.AdmissionReview,
 		Str("name", pod.Name).
 		Logger()
 
-	lctx := logger.
-		WithContext(ctx)
+	lctx := logger.WithContext(context.Background())
```

Comment:
@estahn Oh sorry, regarding the issue, what I meant is that we fixed the mentioned problem by creating a brand new context here, rather than inheriting from the one passed in the function (`ctx`). The PR should now work as expected. My question was about possible use we have of the parent context `ctx` that we might not be aware of. Do you see a reason to add this parent context as a value of the new child context we create here?

And regarding the other PR, as this one will generate a lot of conflicts when merged, we will take care of resolving them in our other PR, not saying we shouldn't merge this one PR, sorry for the confusion.

Reply:
Gotcha. I don't think we need to worry about the context. Thanks for clarifying. 👍🏼